src/tapis-app/src/Sidebar/Sidebar.tsx

```diff
@@ -34,11 +57,45 @@ const SidebarItem: React.FC<SidebarItemProps> = ({ to, label, iconName }) => {
   );
 };
 
+type SidebarGroupProps = {
+  group: string,
+  id: string,
+  label: string,
+  items: SidebarItemProps[]
+};
+
+const SidebarGroup: React.FC<SidebarGroupProps> = ({id, label, items, group}) => {
```

Comment:
Rather than passing in items, can this be the React children property that accepts React nodes instead? The client would then look like:

```
<SidebarGroup>
  <SidebarItem />
  <SidebarItem />
  <SidebarItem />
</SidebarGroup
```

This has the added benefit of not requiring unique id properties.